dir.props

```diff
@@ -45,6 +46,8 @@
     <BuildToolsTaskDir Condition="'$(BuildToolsTargets45)' == 'true'">$(ToolsDir)net45/</BuildToolsTaskDir>
     <UseRoslynCompilers Condition="'$(UseRoslynCompilers)'=='' and '$(OSEnvironment)'!='Windows_NT'">false</UseRoslynCompilers>
   </PropertyGroup>
+
+  <Import Project="$(ToolRuntimePath)BuildVersion.targets" />
```

Comment:
You are right. I named it '.targets' because initially I had targets inside but I removed them, and then I kept it as a .targets file to differentiate from the BuildVersion.props that we generate, but since now we also add the date to the one we generate it probably makes sense to change it to props file now. I'll make the change in BuildTools and then fix it here.